data/wallets.ts

```diff
@@ -0,0 +1,54 @@
+import { Provider, Service } from '../types'
+import { AVAILABLE_FEATURES } from '../helpers/constants'
+import blocto from './wallets/blocto/blocto'
+import dapperWallet from './wallets/dapper-wallet/dapper-wallet'
+import devWallet from './wallets/dev-wallet/dev-wallet'
+import flowWallet from './wallets/flow-wallet/flow-wallet'
+import ledger from './wallets/ledger/ledger'
+import nufi from './wallets/nufi/nufi'
+import shadow from './wallets/shadow/shadow'
+
+type ServiceConfig = Omit<Service, 'provider'> & {
+  provider?: Partial<Provider>
+}
+
+export interface BaseWallet {
+  name: string
+  uid: string
+  icon: string
+  address: string
+  description: string
+  color?: string
+  supportEmail?: string
+  website: string
+  installLink?: {
```

Comment:
Does it matter if the`installLink` and `features` make it so it deviates from the service structure at all?